internal/app/api/handler/index/index.go

```diff
@@ -1,25 +1,33 @@
 package indexhandler
 
 import (
+	"fmt"
 	"io"
 	"net/http"
 	"os"
-
+	"encoding/json"
+	"github.com/globalsign/mgo/bson"
```

Comment:
handler, service layer should not depend on a specific implementation of the repository. If you do this, then what if we change the repository to use a flat file, MySQL or something else,...?

Reply:
Fixed